Libraries/test/TestServerlessApp/SQSEventExamples/ValidSQSEvents.cs.txt

```diff
@@ -0,0 +1,27 @@
+﻿using Amazon.Lambda.Annotations;
+using Amazon.Lambda.Annotations.SQS;
+using Amazon.Lambda.SQSEvents;
+using System;
+using System.Threading.Tasks;
+
+namespace TestServerlessApp.SQSEventExamples
+{
+    public class ValidSQSEvents
```

Comment:
These functions represent valid usage of the `SQSEventAttribute`. However, I have added them in a `.txt` file since I do not want to deploy these function during our integration tests. This file is only in source generator unit tests. Refer to `VerifyValidSQSEvents` unit test added in this PR.

I have added a separate function that will be deployed and used for integ tests. Refer to `TestServerlessApp/SqsMessageProcessing.cs` in this PR.